### PR TITLE
Create idojaras-quarterly-journal-of-the-hungarian-meteorological-ser…

### PR DESCRIPTION
**Pull request by leeladam**
Creating a new style for the Quarterly Journal of the Hungarian Meteorological Service.
http://www.met.hu/en/ismeret-tar/kiadvanyok/idojaras/

### CODE REVIEW

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      <name>Adam Leelossy</name>
+      <email>leeladam@elte.hu</email>
+    </author>
+    <author>
+      <name>Eszter L. Toth</name>
+      <email>toth.eszter@itk.ppke.hu</email>
+    </author>
+    <issn>0324-6329</issn>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: we'd want something other than generic-base here. Probably "science" is the best fit from the available fields?
http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 2]
```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" demote-non-dropping-particle="never" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Id&#x151;j&#xE1;r&#xE1;s - Quarterly Journal of the Hungarian Meteorological Service</title>
+    <title-short>Id&#x151;j&#xE1;r&#xE1;s</title-short>
+    <id>http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service</id>
+    <link rel="self" href="http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service"/>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: did you base this style on an existing style? If so, which? Please include a `rel="template"` link to that style

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+        </else>
+      </choose>
+      <text value="ed"/>
+    </group>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <group font-style="italic" delimiter=", " prefix=" ">
+          <group>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: remove this group. It doesn't do anything.

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      <key macro="issued" sort="descending"/>
+    </sort>
+    <layout font-style="normal">
+      <group suffix=".">
+        <text macro="author" font-style="italic" suffix=","/>
+        <text macro="issued" prefix=" " suffix=":"/>
+        <group prefix=" ">
+          <text macro="title"/>
+          <text macro="container"/>
+          <text macro="locators" font-style="normal"/>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: remove `font-style="normal"`, which doesn't do anything here.

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+          <text macro="event"/>
+          <text macro="publisher"/>
+          <text variable="number-of-pages" prefix=" " suffix=" p."/>
+        </group>
+      </else-if>
+      <else-if type="chapter paper-conference" match="any">
+        <group delimiter=", " prefix=". ">
+          <text macro="event"/>
+          <text macro="publisher"/>
+          <group>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: remove this group (which doesn't do anything)

File: idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+    <updated>2016-05-31T06:37:10+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="container">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" text-case="capitalize-first" prefix=", " suffix=": "/>
+        <group delimiter=", ">
+          <text variable="container-title" text-case="title"/>
+          <text variable="collection-title" text-case="title"/>
```
Comment (adam3smith) [inline idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl]: my sense is that you'd probably want to get rid of all the collection titles. They're not that commonly used, and series (which is what they map to) aren't mentioned in the journal guide.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**